### ISSUE TITLE
NBT data FirstPlayed wrong

### ISSUE THREAD

**Issue by seema84** [PaperSpigot/Paper#7925]
### Expected behavior

The correct timestamp should be stored.

### Observed/Actual behavior

The timestamp for Bukkit/FirstPlayed shows wrong values in NBT Explorer.

An example: a player who has been on the server for at least half a year has the timestamp FirstPlayed: 1653426624076

### Steps/models to reproduce

Since i don't check these timestamps every day, I can't say when or how this happened.

### Plugin and Datapack List

-

### Paper version

This server is running Paper version git-Paper-379 (MC: 1.18.2) (Implementing API version 1.18.2-R0.1-SNAPSHOT) (Git: 276d830)

### Other

It would be very annoying if this data on my almost 10 year old server is now wrong and cannot be fixed.

**Comment by chickeneer** [comment]
I have marked this issue as Cannot reproduce and incomplete. I have searched the relevant code and looked at my own server with NBT Explorer - and cannot identify any sources for this to occur. The data integrity appears correct, as expected. I will offer some explanations on how this may have occurred. In general, there is not enough information for us to explore further.

- The strangest possibility is that a plugin reflected into the server to call CraftPlayer#setFirstPlayed. That seems unlikely, as it is not an API method...
- Maybe even stranger is that the server had the wrong time set (by 5 months) when the player joined - that seems impossibly unlikely.
- More likely explanations. Could be that your server has "multiple" servers, so they may not have connected to this particular server for the first 5 months of them being a member of your server.
- Or something deleted their playerdata file for some reason. This would be a very serious issue - but would need more steps to get to identifying 'why' it was deleted. The provided information does not provide us a direction to investigate it. If there had been an error loading the playerdata file at some point, I suppose this is possible - but that would have caused an error log at some point.

Places you could look for to get more information. 
Look to see what happened around (and before) May 24th when their player file seems to think they connected for the first time.
Search for logs or time values of them connecting before the date you have indicated.

We have no way to proceed on this issue. Thank you for submitting this report even if we can't help you yet.

**Comment by seema84** [comment]
Thank you very much for the feedback.

Maybe I can point out some more information for the problem.
I do not use a BungeeCord/Waterfall server. There are also no multiple servers. I can actually rule that out.

What I noticed is that it doesn't affect all accounts.
Some are not affected at all, some are affected and some are affected a lot so it changes almost daily.

I noticed in the logs that this change happens when the affected players log out.
